Dockerfile-debian

```diff
@@ -0,0 +1,39 @@
+# SPDX-FileCopyrightText: 2021 Free Software Foundation Europe e.V. <https://fsfe.org>
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# Like normal Dockerfile, but based on python:slim (Debian) to ease compliance
+
+# Create a base image that has dependencies installed.
+FROM python:slim-buster AS base
+
+RUN apt-get update \
+    && apt-get install -y git mercurial python3 \
```

Comment:
I'm pretty confident that when you start from `python` base image you do *not* need to install python ;-)
This will save you some disk space (due to all the dependencies you are currently installing)

Reply:
Whoops, my bad. Indeed, it saves ~30MB disk space